test/runtests.jl

```diff
@@ -129,7 +129,8 @@ import Bcube:
     Bar2_t,
     Tri3_t,
     Quad4_t,
-    Quad9_t
+    Quad9_t,
+    foreach_element
```

Comment:
congrats for the numerous tests! Could you add one for `map_element`?

Reply:
done